### ISSUE TITLE
2.7.6.3 - _winreg.EnumKey extremely slow

### ISSUE THREAD

**Issue by 2-5** [IronLanguages/main#1400]
After upgrading to 2.7.6.3, this script is extremely slow (compared with IronPython 2.7.5):

```
import mimetypes
mimetypes.add_type("application/json", ".json")
```

The underlying problem seems to be that `_winreg.EnumKey` is very slow:

```
import _winreg

with _winreg.OpenKey(_winreg.HKEY_CLASSES_ROOT, '') as hkcr:
    i = 0
    while True:
        try:
            print("{} {}".format(i, _winreg.EnumKey(hkcr, i)))
        except EnvironmentError:
            break
        i += 1
```

**Comment by slide** [comment]
Thanks for reporting this. I'll take a look soon.

**Comment by slide** [comment]
Fixed with #1428
